docker/final.Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN apt-get -q update \
      python3-git \
      python3-minimal \
      python3-semantic-version \
-     python3-websocket \
+     python3-websockets \
```

Comment:
Thanks for testing and fixing!

I think we need to keep `websocket`, too, as [zeek-client uses it](https://github.com/zeek/zeek-client/blob/master/zeekclient/controller.py#L8).